equinoxfitness/robopager
# DATCORE-1004 Fix pyyaml security severity

update setup.py for pyyaml


### How Has This Been Tested?





1. Checkout this branch

1. Switch to virtual environment

1. Run `python -m run`

- [x] Verify run is successful


### Rollback Plan
